proto/Cargo.toml

```diff
@@ -27,9 +27,10 @@ protobuf = "2"
 lazy_static = { version = "1.3", optional = true }
 
 [build-dependencies]
-protobuf-build = "0.7"
+protobuf-build = { version = "0.8", default-features = false }
 grpcio-compiler = { path = "../compiler", version = "0.5.0-alpha.2", default-features = false }
 prost = { version = "0.5", optional = true }
 prost-types = { version = "0.5", optional = true }
 protobuf = { version = "2", optional = true }
```

Comment:
I think all these optional build deps can be removed

Reply:
You are right.